controllers/apply/awards-for-all/transforms.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+const config = require('config');
+const has = require('lodash/fp/has');
+const get = require('lodash/fp/get');
+const Sentry = require('@sentry/node');
+
+const logger = require('../../../common/logger').child({
+    service: 'transform'
+});
+
+function transformProjectDateRange(applicationData) {
+    if (has('projectDateRange')(applicationData)) {
+        logger.info('Transforming projectDateRange');
+
+        try {
+            const oldDateRange = get('projectDateRange')(applicationData);
+            applicationData.projectStartDate = oldDateRange.startDate;
+            applicationData.projectEndDate = oldDateRange.endDate;
+            delete applicationData['projectDateRange'];
+        } catch (err) {
+            logger.error('Failed to transform projectDateRange');
+            Sentry.captureException(err);
+        }
+    }
+
+    return applicationData;
+}
+
+function transform(applicationData) {
+    // Add additional transforms here
+    if (config.get('awardsForAll.enableNewDateRange') === true) {
+        return transformProjectDateRange(applicationData);
```

Comment:
Just to check, we're calling this like so:
```
    const originalData = application.applicationData || {};
    const data = transform(originalData);
```

This means it'll modify the original `application.applicationData` too, right? Probably okay in this case but is it worth cloning the source data here before the transforms?

Reply:
It shouldn't do as I'm passing it in separately, but will double check.